jaxb-ri/codemodel/codemodel/src/test/java/com/sun/codemodel/Issue1505Test.java

```diff
@@ -0,0 +1,127 @@
+/*
+ * Copyright (c) 2021 Oracle and/or its affiliates. All rights reserved.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Distribution License v. 1.0, which is available at
+ * http://www.eclipse.org/org/documents/edl-v10.php.
+ *
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+package com.sun.codemodel;
+
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+
+import org.junit.Ignore;
+
+public class Issue1505Test {
+    
+    private void checks(String test) throws ClassNotFoundException {
+        checks(test, test);
+    }
+    
+    private void checks(String expected, String test) throws ClassNotFoundException {
```

Comment:
can throwing checked exception from this method be avoided  by throwing out either some assertionerror or other runtime/unchecked exception if cnfe happens? It does not make sense to let tests re-throw it